addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -443,11 +444,27 @@ class MessageList extends Component {
             return false;
         }
         const loadMoreRect = loadMore.getBoundingClientRect();
-        const elRect = this.el.getBoundingClientRect();
+        const elRect = this.scrollableParent.getBoundingClientRect();
         const isInvisible = loadMoreRect.top > elRect.bottom || loadMoreRect.bottom < elRect.top;
         return !isInvisible;
     }
 
+    /**
+     * @private
+     * @param {Element}
+     * @returns {Element} closest scrollable node element
+     */
+    getScrollableParent(node) {
+        if (node == null) {
```

Comment:
Never use double `==`, but I'm not sure we want to check for `null` specifically anyway.
Just `!node` should be good here?

But see comment below, this can probably be removed.